### DISCUSSION TITLE
Adding a new dependency for a plugin

### DISCUSSION THREAD

**Discussion by surmish** [LazyVim/LazyVim#2476] [Q&A] [answered]
The LazyVim telescope configuration is like so:

![image](https://github.com/LazyVim/LazyVim/assets/33273194/4f87f368-519d-48ba-af00-4b081b9c286d)

How do I add a dependency to `debugloop/telescope-undo.nvim` in my `editor.lua` without overwriting the LazyVim configuration? I also don't want to copy over LazyVim telescope config entirely into my personal config files.

**Comment by dpetka2001** [comment, marked as answer]
Please read the [docs](https://www.lazyvim.org/configuration/plugins#%EF%B8%8F-customizing-plugin-specs). Your custom `dependencies` will be merged with the Lazyvim `dependencies`. So just create a file in `/lua/plugins/` directory and put there your custom Telescope spec with the `dependencies` you want.
